python/paddle/v2/framework/layers.py

```diff
@@ -364,12 +362,12 @@ def conv2d(input,
 
 
 def sequence_pool(input, pool_type, **kwargs):
-    ENUM_POOL_TYPE = set(["MAX", "AVG", "SQRT", "LAST", "FIRST"])
+    ENUM_POOL_TYPE = set(["MAX", "SUM", "AVERAGE", "SQRT", "LAST", "FIRST"])
```

Comment:
AVERAGE  => 'AVG'?

Reply:
It's weird implemented in sequence_pool operator. Because it has been used in chapter5 and the chapter6 in coming, so I will fix it after chapter6 and other sequence PRs merged.
Otherwise, it will make our in progress things into a mess.